content/applications/websites/website/pages.rst

```diff
@@ -1,9 +1,186 @@
+- :guilabel:`Indexed`: Disable if you don't want the page to be indexed, i.e., shown in search
+  engine results;
+- :guilabel:`Published`: Enable to publish the page;
+- :guilabel:`Publishing Date`: To publish the page at a specific moment, select the date,
+  click the clock icon to set the time, then click the green check mark to validate your selection.
+- :guilabel:`Visibility`: Select who can access the page:
+
+  - :guilabel:`All`
+  - :guilabel:`Signed In`
+  - :guilabel:`Restricted Group`: Select the group in the :guilabel:`Authorized group` field.
+  - :guilabel:`With Password` : Enter the password in the :guilabel:`Password` field.
+
+.. tip::
+   Alternatively, *some* of these properties can also be modified from :menuselection:`Website -->
+   Site --> Pages`.
+
+Duplicating pages
+~~~~~~~~~~~~~~~~~
+
+To duplicate a page, access the page, then go to :menuselection:`Website --> Site --> Properties`
+and click :guilabel:`Duplicate Page`. Enter a :guilabel:`Page Name`, then click :guilabel:`OK`.
+By default, the new page is added at the end of the menu, but you can remove it from the menu or
+move its position using the menu editor.
+
+.. _website/delete-page:
+
+Deleting pages
+~~~~~~~~~~~~~~
+
+To delete a page, proceed as follows:
+
+#. Access the page, then go to :menuselection:`Website --> Site --> Properties` and click
+   :guilabel:`Delete Page`.
+#. A pop-up window appears on the screen with all links referring to the page you want to delete,
+   organized by category. Expand a category, e.g., :guilabel:`Page`, then click on a link to open
+   it in a new window.
+#. Once you have updated the links (or set up a :ref:`redirection <website/URL-redirection>`),
+   select the :guilabel:`I am sure about this` check box, then click :guilabel:`OK`.
+
+.. note::
+   Visitors are redirected to a 404 error page when they try to access a deleted page, either
+   directly or through a link that hasn't been updated. You can prevent this by defining a
+   :ref:`redirection <website/URL-redirection>` for the page's URL.
+
+.. _website/URL-redirection:
+
+URL redirects
+-------------
+
+Redirecting URLs consists in sending visitors and search engines to a URL that is different from the
+one they originally requested. This technique is used, for example, to prevent broken links when you
+:ref:`delete a page <website/delete-page>`, :ref:`modify its URL <website/page_properties>`, or move
+your site to a new :doc:`domain </administration/maintain/domain_names>`. It can also be used to
+improve :doc:`pages/seo`.
+
+To access existing URL redirections for your website and create new ones, :doc:`activate the
+developer mode </applications/general/developer_mode>` and go to :menuselection:`Website -->
+Configuration --> Redirects`.
+
+.. note::
+   A record is added automatically everytime you :ref:`modify a page's URL <website/page_properties>`
+   and enable :guilabel:`Redirect Old URL`.
```

Comment:
You can seperate at any "space". Here's the only suggestion for which I checked the number of characters x) 
```suggestion
.. note::
   A record is added automatically everytime you :ref:`modify a page's URL
   <website/page_properties>` and enable :guilabel:`Redirect Old URL`.
```